renderer/views/player.js

```diff
@@ -540,3 +548,37 @@ function cssBackgroundImageDarkGradient () {
   return 'radial-gradient(circle at center, ' +
     'rgba(0,0,0,0.4) 0%, rgba(0,0,0,1) 100%)'
 }
+
+// Format the playback time of the video
+function formatPlaybackProgress (currentTime, duration) {
```

Comment:
I think the reason why I did both `currentTime` and `duration` together was to make sure that the time format for both would be the same. Otherwise this could lead to something like `01:02 / 02:20:47` when the user is playing, which can be confusing, compared to `00:01:02 / 02:20:47`.